postman/Sila API v0.2 - Local Signer Server Edition.postman_collection.json

```diff
@@ -5545,6 +5630,437 @@
 				}
 			]
 		},
+		{
+			"name": "Virtual Accounts",
+			"item": [
+				{
+					"name": "Open Virtual Account",
+					"request": {
```

Comment:
It looks like there's already a `Virtual Accounts` folder in this collection. Is that intentional?

Reply:
ah, likely not.  let me fix

Reply:
it's a dupe from a merge :-/  what a mess